README.md

```diff
@@ -145,15 +198,33 @@ the configuration. If conventions need to be applied in a certain order
 (which should be avoided if at all possible), it should be called out
 in their respective READMEs.
 
+Follow any configuration changes with the "Update" sequence described below:
+
 ### Update
 
-Periodically, run `make boilerplate-update` on a clean branch in your
-consuming repository. If it succeeds, commit the changes, being sure to
-notice if any new files were created. **Sanity check the changes against
-your specific repository to ensure they didn't break anything.** If they
-did, please make every effort to fix the issue _in the boilerplate repo
-itself_ before resorting to local snowflake fixups (which will be
-overwritten the next time you update) or opting out of the convention.
+Use this procedure to pick up newly-subscribed conventions; and run it
+periodically to pick up changes to existing subscriptions or to the
+boilerplate framework itself.
+
+1. Run `make boilerplate-update` on a clean branch in your consuming
+   repository.
+
+2. Run `make boilerplate-commit`. This will create a new topic branch
+   and commit any changes resulting from the update.
+
+3. Sanity check the changes against your specific repository, fixing any
+   breakages and making local changes appropriate to the substance of
+   the update. If you used `make boilerplate-commit`, you can use
+   `git show` to see a summary of what was changed. **NOTE:** You must
+   not touch files owned by boilerplate. Any changes to boilerplate
+   content must be made in the boilerplate repo itself.
+
+4. If local changes were necessary, commit them manually. You should
```

Comment:
nit : wording between points 3 and 4 sounds a bit odd (one considering we use 'boilerplate-commit' while this is an alternative in 3)

Reply:
Ack, I'll play with it.

Reply:
Done.